scripts/import_cldr.py

```diff
@@ -755,14 +755,24 @@ def parse_currency_names(data, tree):
 
 def parse_unit_patterns(data, tree):
     unit_patterns = data.setdefault('unit_patterns', {})
+    compound_patterns = data.setdefault('compound_unit_patterns', {})
     for elem in tree.findall('.//units/unitLength'):
         unit_length_type = elem.attrib['type']
         for unit in elem.findall('unit'):
             unit_type = unit.attrib['type']
+            box = "%s:%s" % (unit_type, unit_length_type)
```

Comment:
For readability, I'd rename:
- box -> unit_and_length and
- pattern_box -> unit_and_length_patterns

Don't care much, but please do at the very least rename
- pattern_box -> box_patterns